charts/mimir-distributed/templates/gateway/gateway-pdb.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.enterprise.enabled -}}
```

Comment:
Does the gateway need a PDB? Failed writes will buffer and retry, failed queries can just be retried. Not sure I see the need like I do with the ingesters/admin api.

Reply:
I added it just for consistency, maybe no-one will actually set it